flink-formats/flink-avro-confluent-registry/src/test/java/org/apache/flink/formats/avro/registry/confluent/CachedSchemaCoderProviderTest.java

```diff
@@ -22,7 +22,7 @@
 import io.confluent.kafka.schemaregistry.client.rest.RestService;
 import io.confluent.kafka.schemaregistry.client.security.basicauth.BasicAuthCredentialProvider;
 import io.confluent.kafka.schemaregistry.client.security.bearerauth.BearerAuthCredentialProvider;
-import org.junit.Test;
+import org.junit.jupiter.api.Test;
 import org.powermock.reflect.Whitebox;
```

Comment:
It's tricky to get rid of this `Whitebox` ... I left it for now.

I'm thinking the best alternative might actually be to remove a bunch of these tests: SSL setup is tested extensively internally to the Confluent Schema Registry.  What do you think?